python/pylibcugraph/pylibcugraph/device_array_pybuffer.pyx

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
Thanks for noticing that, no it isn't being used anywhere so I removed it in the latest commit.

However, we did manage to remove a device-to-device copy in PLC's `uniform_neighbor_sample` by returning a CuPy array "view" instead of the old approach of copying the data to a new owning CuPy array [here](https://github.com/rapidsai/cugraph/pull/2684). That resulted in a small speedup (as shown by the before-after benchmarks in that PR) and the elimination of a memory spike while making the copy. The `DeviceArrayPyBuffer` was an earlier attempt to do the same thing, but I never removed that class after switching to the CuPy array approach.